testData/parser/GlobalVariables.txt

```diff
@@ -0,0 +1,29 @@
+Matlab File
+  MatlabGlobalBlockImpl(GLOBAL_BLOCK)
+    PsiElement(global)('global')
+    PsiWhiteSpace(' ')
+    PsiElement(IDENTIFIER)('a')
+    PsiWhiteSpace(' ')
+    PsiElement(IDENTIFIER)('b')
+    PsiWhiteSpace(' ')
+    PsiElement(IDENTIFIER)('c')
+  PsiElement(;)(';')
+  PsiElement(NEWLINE)('\n')
+  PsiElement(NEWLINE)('\n')
+  MatlabGlobalBlockImpl(GLOBAL_BLOCK)
+    PsiElement(global)('global')
+    PsiWhiteSpace(' ')
+    PsiElement(IDENTIFIER)('d')
+    PsiWhiteSpace(' ')
+    PsiElement(...)('...')
+    PsiElement(NEWLINE)('\n')
+    PsiWhiteSpace('    ')
+    PsiElement(IDENTIFIER)('e')
+    PsiWhiteSpace(' ')
+    PsiElement(...)('...')
+    PsiElement(NEWLINE)('\n')
+    PsiWhiteSpace('    ')
+    PsiElement(IDENTIFIER)('f')
+    PsiWhiteSpace(' ')
+    PsiElement(IDENTIFIER)('g')
+    PsiElement(NEWLINE)('\n')
```

Comment:
Seems like tests should trim extra whitespaces in test data
When I run the test I see that there is no last line `PsiElement(NEWLINE)('\n')`, so just remove it (and re-run test to check that it works)